### ISSUE TITLE
[Geometry tasks] Combine "draw" and "capture" geometry tasks

### ISSUE THREAD

**Issue by gino-m** [google/ground-platform#1649]
@lecrabe @rawbzz @vittorino @amegantz 

One idea for future work to further reduce confusion around "Capture location", "Drop a pin", and "Draw/walk perimeter". What if we always capture location (with user's permission) as well as the user-defined location? Then it can be up to the analyst which to use - the actual device location or the user-defined location on the map, if they different. 

We could even show something on the map like a dotted line between your current physical location, and the selected location + distance in ft/m.

**Comment by amegantz** [comment]
If there aren't any bandwidth issues with the constant capturing, I think this more automatic approach makes sense. Maybe this could be the default, but we could still give users the option to complete tasks manually in case they don't consent to the constant capturing or have bandwidth/connection issues

**Comment by gino-m** [comment]
> If there aren't any bandwidth issues with the constant capturing, I think this more automatic approach makes sense. Maybe this could be the default, but we could still give users the option to complete tasks manually in case they don't consent to the constant capturing or have bandwidth/connection issues

No issues with resource usage - we would only record to location when the user adds the point or adds a vertex.

Re permissions, for all the projects I've seen the user's actual location is essential. We should make it super clear that we're capturing the user's location so that they can avoid collecting data if they disagree, but as far as field data collection I think making it optional is a low priority.

**Comment by gino-m** [comment]
We discussed this further at our weekly PM/UX sync. It turns out that the user may not want to drop a pin or polygon vertex at the same moment they want to capture location, and representing captured location for adds additional UX complications. Instead, we proposed leaving these as distinct tasks, but always adding a "Capture location" task by default (#1735), and show the three geometry annotation tasks under "advanced" (#1736).
